Sources/Mini/ActionReducer.swift

```diff
@@ -1,35 +1,19 @@
-/*
- Copyright [2019] [BQ]
```

Comment:
Keep the copyright and the license in every file as it was before. Update the license holder to HyperDevs since 2021.

Reply:
Fixed.

Reply:
Fixed!